README.md

```diff
@@ -2,3 +2,12 @@ fluentd-catch-all-config
 ========================
 
 Collection of configuration files for the Fluentd log collection agent. Intended to enable the automagic collection of most logs generated by many of the most popular applications run in the cloud.
+
+Major version updates may introduce changes that are not backwards compatible.
```

Comment:
Just realized -- let's bold this, to make sure people see this right away.

Reply:
Done.